### ISSUE TITLE
No focus outline on the number input when using keyboard navigation in the MetaMask extension swaps feature

### ISSUE THREAD

**Issue by georgewrmarshall** [georgewrmarshall/metamask-extension#8]
### Describe the bug
When using the MetaMask extension swaps feature with keyboard navigation, there is no focus outline on the number input.

### Browser
Brave

### Hardware Wallet
None

### Operating System
macOS

### Version
v12.0.1

### Steps to Reproduce
1. Open MetaMask on Brave.
2. From the homepage, click the swaps action button.
3. Navigate to the swaps number input using the keyboard.
4. Notice that the input field doesn't contain a focus outline.

### Additional Context
No additional context provided.
